src/test/java/org/traccar/protocol/MegastekProtocolDecoderTest.java

```diff
@@ -50,9 +51,20 @@ public void testDecode() throws Exception {
                 "0125$MGV002,860719020193193,DeviceName,R,240214,104742,A,2238.20471,N,11401.97967,E,00,03,00,1.20,0.462,356.23,137.9,1.5,460,07,262C,0F54,25,0000,0000,0,0,0,28.5,28.3,,,100,Timer;"));
 
         verifyPosition(decoder, text(
-                "$MGV002,860719020193193,DeviceName,R,240214,104742,A,2238.20471,N,11401.97967,E,00,03,00,1.20,0.462,356.23,137.9,1.5,460,07,262C,0F54,25,0000,0000,0,0,0,28.5,28.3,,,100,Timer;!"),
```

Comment:
Was this sample removed?

Reply:
That was mostly a mistake by me. I restored the old line.